packages/catalog-search/src/SearchSuggestions.jsx

```diff
@@ -25,6 +30,9 @@ const SearchSuggestions = ({ autoCompleteHits, enterpriseSlug, handleViewAllClic
                 highlightedTitle={hit._highlightResult.title.value}
                 authoringOrganization={hit.key && hit.key.split('+')[0]}
                 title={hit.title}
+                hit={hit}
```

Comment:
upto you this one, but given we are passing `hit` here now, seems like passing in other props like authoringOrg and title that depend on hit, may be superflous ... but if it's too much refactor we can hold back on changing this

Reply:
I had the same thought when adding hit, I can definitely refactor